fmn/rules/anitya.py

```diff
@@ -35,6 +35,36 @@ def anitya_catchall(config, message):
     return message['topic'].split('.')[3] == 'anitya'
 
 
+@hint(categories=['anitya'], invertible=False)
+def anitya_specific_distro(config, message, distro=None, *args, **kw):
+    """ Distro-specific release-monitoring.org events
+
+    This rule will match all anitya events *only for a particular distro*.
+    """
+
+    if not distro:
+        return False
+
+    if not anitya_catchall(config, message):
+        return False
+
+    d = message['msg'].get('distro', {})
+    if d:  # Have to be careful for None here
+        if d.get('name') == distro:
```

Comment:
Do we do ``.lower()`` comparisons of distro names elsewhere?  If we do, then it would make sense to do so here so that we're all symmetric.